cli/src/main/java/io/quarkiverse/argocd/cli/application/GenerationBaseCommand.java

```diff
@@ -35,12 +35,16 @@ public abstract class GenerationBaseCommand extends ApplicationBaseCommand imple
     @Option(names = { "-n", "--namespace" }, description = "The target namespace")
     Optional<String> namespace = Optional.empty();
 
+    @Option(names = { "--app-namespace" }, description = "The target namespace")
```

Comment:
Description here should be different from `-n | --namespace` to make things clear